GeneratorInterface/GenFilters/BuildFile.xml

```diff
@@ -1,30 +1,3 @@
-<use   name="fastjet"/>
-<use   name="boost"/>
-<use   name="FWCore/PluginManager"/>
-<use   name="FWCore/ParameterSet"/>
-<use   name="FWCore/Framework"/>
-<use   name="FWCore/Utilities"/>
-<use   name="SimDataFormats/GeneratorProducts"/>
-<use   name="heppdt"/>
-<use   name="clhep"/>
-<use   name="root"/>
-<use   name="GeneratorInterface/Pythia6Interface"/>
-<use   name="GeneratorInterface/Core"/>
-<use   name="pythia8"/>
-<use   name="SimGeneral/HepPDTRecord"/>
-<use   name="DataFormats/GeometryVector"/>
-<use   name="DataFormats/GeometrySurface"/>
-<use   name="TrackPropagation/SteppingHelixPropagator"/>
-<use   name="MagneticField/Records"/>
-<use   name="TrackingTools/TrajectoryState"/>
-<use   name="TrackingTools/TrajectoryParametrization"/>
-<use   name="TrackingTools/Records"/>
-<use   name="CommonTools/UtilAlgos"/>
-<use   name="FWCore/ServiceRegistry"/>
-<use   name="CommonTools/BaseParticlePropagator"/>
-<use   name="TrackingTools/GeomPropagators"/>
-<use   name="DataFormats/HepMCCandidate"/>
-<use   name="DataFormats/JetReco"/>
-<use   name="DataFormats/EgammaReco"/>
-<use   name="DataFormats/Math"/>
-<flags   EDM_PLUGIN="1"/>
+<export>
+  <lib name="1"/>
+</export>
```

Comment:
@fabiocos , there is no need to have this BuildFile. BuildFile in package directory is only needed when there are source files in package/{src,interface} directory.

Reply:
@smuzaffar ok, thanks for the remark